app/facades/books/index_facade.rb

```diff
@@ -24,6 +24,10 @@ def categories
       Book.categories
     end
 
+    def book_presence
```

Comment:
this method returns true or false, so it is better to name it with a question mark at the end. For example, `books_present?`